frontend/magic_links/filters.py

```diff
@@ -1,16 +1,31 @@
 from datetime import datetime
 
+from flask_babel import gettext
 
-def datetime_format(value):
-    date_format = "%d %B %Y at %H:%M"
-    am_pm_format = "%p"
-    formatted_time = datetime.strptime(value, "%Y-%m-%d %X").strftime(
-        date_format
-    )
-    formatted_time = (
-        formatted_time
-        + datetime.strptime(value, "%Y-%m-%d %X")
-        .strftime(am_pm_format)
-        .lower()
-    )
-    return formatted_time
+
+def _month_from_number(month_number: int) -> str:
+    number_to_full_month = {
+        1: gettext("January"),
+        2: gettext("February"),
+        3: gettext("March"),
+        4: gettext("April"),
+        5: gettext("May"),
+        6: gettext("June"),
+        7: gettext("July"),
+        8: gettext("August"),
+        9: gettext("September"),
+        10: gettext("October"),
+        11: gettext("November"),
+        12: gettext("December"),
+    }
```

Comment:
This could alternatively have been a hard-coded `Mapping` on the module & pulled back with: [get_lang.py#L5](https://github.com/communitiesuk/funding-service-design-utils/blob/c0d25d6bebf13c9f0efe32f250dd1f6931e1500a/fsd_utils/locale_selector/get_lang.py#L5)

i.e
```
{
    "en" : {
        1: "January"
        ...
    }, 
    "cy": {
        1: "Ionawr"
        ...
    }
}
```

The reason I went against this was so we didn't have to maintain two ways of translation, it seems we're using babel so we can leverage `gettext` (like in this example) and follow the convention like the rest of the application.

Reply:
Could we check if we can do this using native datetime support in Babel i.e. https://babel.pocoo.org/en/latest/api/dates.html rather than custom logic?

Reply:
Good point I'll take a look

Reply:
That's much better, thanks for mentioning that, have added it in https://github.com/communitiesuk/funding-service-design-authenticator/pull/95/commits/89bfcbd9e949bada7e0947c411984ebd3711d269